proposal/color-4-new-spaces.md

```diff
@@ -0,0 +1,1984 @@
+  > We can compare, but we can't do conversion. The color space remains
+  > relevant to equality. While this is technically the same as using `==`,
+  > it makes the function more robust to allow comparison of all colors.
+
+* Otherwise:
+
+  * Let `color1` and `color2` be the result of [converting] `$color1` and
+    `$color2` into `xyz` color space, respectively.
+
+* Return `color1 == color2`.
+
+## Modified Color Module Functions
+
+### `color.hwb()`
+
+These functions are now deprecated. Authors should use global `hwb()` instead.
+
+* ```
+  hwb($channels)
+  ```
+
+  * Return the result of calling the global function `hwb($channels)`.
+
+* ```
+  hwb($hue, $whiteness, $blackness, $alpha: 1)
+  ```
+
+  * Return the result of calling the global function
+    `hwb(list.slash($hue $whiteness $blackness, $alpha))`.
+
+### `color.mix()`
+
+```
+mix($color1, $color2,
+  $weight: 50%,
+  $method: null)
+```
+
+* If either `$color1` or `$color2` is not a color, throw an error.
+
+* If `$method` is null:
+
+  * If either `$color1` or `$color2` is not a [legacy color], throw an error.
+
+    > Method is required for non-legacy colors. This matches the `color-mix()`
+    > function defined in [Colors Level 5][color-5], and allows us to add
+    > additional default behavior in the future.
+
+  * Return the result of [legacy interpolation] between `$color1` and `$color2`
+    with the specified `$weight`.
+
+* Otherwise, if `$method` is not a [color interpolation method], throw an error.
+
+* Return the result of [interpolating] between `$color1` and
+  `$color2` with the specified `$weight` and `$method`.
+
+### `color.change()`
+
+```
+change($color, $args...)
+```
+
+This function is also available as a global function named `change-color()`.
+
+* If `$color` is not a color, throw an error.
+
+* If any item in `$args` is not a keyword argument, throw an error.
+
+* Let `origin-space` be `$color`'s color space.
+
+* If the keyword argument `$space` is specified in `$args`:
+
+    * Let `space` be the result of [looking up a known color space] named
+      `$space`.
+
+    * If `space` is not a [known color space], and `space != origin-space`,
+      throw an error.
+
+      > We can make changes in an unknown space, but we can't do conversion.
+
+    * Let `color` be the result of [converting] `$color` to `space`.
+
+* Otherwise:
+
+  * Let `space` be `origin-space`.
+
+  * Let `color` be the value of `$color`.
+
+* Let `alpha` be `color`'s alpha property.
+
+* If the keyword argument `$alpha` is specified in `$args`:
+
+    * Set `alpha` to the result of [percent-converting] `$alpha`, and clamping
+      it between 0 and 1 (inclusive).
+
+* Let `channel-args` be the remaining keyword arguments in `$args`, not
+  including `$space` or `$alpha` arguments.
+
+* Let `channels` be a list of the `color`'s channels.
+
+* For each keyword `key` and value `new` in `channel-args`:
+
+  * If `new` is not a number or the special value `none`, throw an error.
```

Comment:
Same comment as above: we need to be explicit about how to recognize a SassScript value that indicates `none`.